src/pages/chat.tsx

```diff
@@ -57,105 +100,39 @@ const ChatPage: NextPage = (props): JSX.Element => {
-                                stress
-                            </ChatBox>
-                        </Chat>
-
-                        <Chat user="receiver">
-                            <ChatBox>
-                                Oh no! I'm so sorry to hear that. Have you tried
-                                contacting your instructors?
-                            </ChatBox>
-                        </Chat>
-
-                        <Chat user="sender">
-                            <ChatBox>
-                                No I have not, I don't think they are going to
-                                do anything
-                            </ChatBox>
-                        </Chat>
-
-                        <Chat user="receiver">
-                            <ChatBox>
-                                I can understand why you might feel that way. I
-                                used to be the same during my freshman year.
-                            </ChatBox>
-                            <ChatBox>
-                                But I started letting them know when I was in
-                                difficult circumstances and they were very
-                                understanding and helpful, you should definitely
-                                send them an email and explain your situation
-                            </ChatBox>
-                            <ChatBox>
-                                There's nothing to lose by doing that!
-                            </ChatBox>
-                        </Chat>
-
-                        <Chat user="sender">
-                            <ChatBox>Really?</ChatBox>
-                            <ChatBox>
-                                I had the assumption that professors at UK
-                                wouldn't be understanding
-                            </ChatBox>
-                        </Chat>
-
-                        <Chat user="receiver">
-                            <ChatBox>
-                                Yes! Many friends I know also had that
-                                assumption until they contacted their professors
-                            </ChatBox>
-                        </Chat>
-
-                        <Chat user="sender">
-                            <ChatBox>Got youuu?</ChatBox>
-                            <ChatBox>Thank you so much!</ChatBox>
-                            <ChatBox>
-                                I am going to try doing that, you have a good
-                                day.
-                            </ChatBox>
-                        </Chat>
+                        {/* If a message exists, add the latest message to the chat window */}
+                        {messages &&
+                            messages.map((msg, index) => (
+                                <Chat
+                                    user={
+                                        index % 2 == 0 ? 'sender' : 'receiver'
```

Comment:
We don't want these alternating, do we? Or is this just for demo purposes?

Reply:
As discussed over discord, will be leaving this for demonstration purposes. Will be handled in sending/receiving messages tickets.